### PR TITLE
Bugfix #2400

### PR DESCRIPTION
**Pull request by vladi-j**
Issue: #2400 

1. Samples title is displayed only if samples exist
2. Instead of star, text: "This field is required!" is displayed
3. Negotiated to implement indent as in luma - big indent and not bold link
4. Link option is in line with other elements

### CODE REVIEW

File: packages/scandipwa/src/component/ProductDownloadableLinks/ProductDownloadableLinks.component.js [hunk 1]
```diff
@@ -25,22 +25,34 @@ export class ProductDownloadableLinks extends PureComponent {
...
     };
 
     getLabel(link) {
         const { title, price } = link;
         const { isRequired } = this.props;
 
         if (!isRequired) {
-            return title;
+            return (
+
```
Comment (zans-laksa) [inline packages/scandipwa/src/component/ProductDownloadableLinks/ProductDownloadableLinks.component.js:43]: Please remove empty line.

File: packages/scandipwa/src/component/ProductDownloadableLinks/ProductDownloadableLinks.component.js [hunk 2]
```diff
@@ -89,17 +101,38 @@ export class ProductDownloadableLinks extends PureComponent {
...
+            </div>
+        );
+    }
+
+    renderLinks() {
+        const { links, isRequired } = this.props;
+        return (
+            <>
+            { links.map(this.renderLink) }
+            { this.renderRequired(isRequired) }
```
Comment (zans-laksa) [inline packages/scandipwa/src/component/ProductDownloadableLinks/ProductDownloadableLinks.component.js:126]: Please indent this.

File: packages/scandipwa/src/component/ProductDownloadableLinks/ProductDownloadableLinks.style.scss [hunk 3]
```diff
@@ -54,6 +60,11 @@
         }
     }
 
+    &-Required {
+        color: var(--primary-error-color);
+        padding: 1rem 0 0rem 1.5rem;
```
Comment (zans-laksa) [inline packages/scandipwa/src/component/ProductDownloadableLinks/ProductDownloadableLinks.style.scss:65]: Please replace 0rem with 0.

**Review by zans-laksa — CHANGES_REQUESTED** [review]
Overall good. Please fix some minor issues commented inline.

**Comment by vladi-j** [comment]
Minor issues were fixed.

**Review by zans-laksa — APPROVED** [review]
Code review passed.

**Pull request merged**